pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,41 @@ title: Reduce bandwidth usage
 
-In order to give our power users control over data flow, we enabled a few `BALENA_` variables in the [configuration][configuration] tab.
+Most of these controls are available from the fleet/device configuration tabs on the dashboard as well as via Supervisor variables. The NetworkManager connectivity check may be configured via `config.json`.
 
-| Variable                               | Allowed Value      | Action                                             | Default | Available with Supervisor Version |
-|----------------------------------------|--------------------|----------------------------------------------------|---------|-----------------------------------|
-| `BALENA_SUPERVISOR_VPN_CONTROL`        | true/false         | Enable / Disable cloudlink                         | true    | v1.1.0                            |
-| `BALENA_SUPERVISOR_CONNECTIVITY_CHECK` | true/false         | Enable / Disable cloudlink connectivity check      | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_POLL_INTERVAL`      | 600000 to 86400000 | API Poll interval in milliseconds                  | 900000  | v1.3.0                            |
-| `BALENA_SUPERVISOR_LOG_CONTROL`        | true/false         | Enable / Disable logs from being sent to API       | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_HARDWARE_METRICS`   | true/false         | Enable / Disable device hardware metrics reporting | true    | v12.8.0                           |
+**The minimum bandwidth required is approximately 1 MB per month**, when turning off all of the features below and setting API Polling to the minimum value of once per day. These measurements were made with a Quectel modem on a SixFab HAT on a Raspberry Pi4, and using a Twilio North American SIM. See this [blog post](https://www.balena.io/blog/cellular-iot-isnt-as-hard-as-you-think/) for a similar setup. The device was running balenaOS v2.107.10 and Supervisor v14.4.4.
 
-Side Effects / Warnings
--------------------
+| Control | Monthly Bandwidth if enabled | Variable Name<br>`BALENA_SUPERVISOR...` | Comments |
+| ------- | --------- | -------- | -------- |
+| Cloudlink VPN | 81 MB | ...`_VPN_CONTROL` | Bandwidth just to maintain connection |
+| Device Metrics | 73 MB | ...`_HARDWARE_METRICS` | Minimal application activity |
+| API Poll Interval, 4x hour | 28 MB | ...`_POLL_INTERVAL` | Default interval |
+| Network Manager connectivity check, 1x hour | 8 MB | No variable, see [configuration][config-json] to set interval | Default interval |
+| API Poll Interval, 1x day | 1 MB | ...`_POLL_INTERVAL` | Minimum interval |
+| Device Logging| ?? | ...`_LOG_CONTROL` | Minimal application activity |
+| Cloudlink connectivity check | N/A | ...`_CONNECTIVITY_CHECK`  | Only active when VPN is down |
 
```

Comment:
Future visions: How are we planning to keep the data updated? Do we have a test environment spun up somewhere that can keep this table updated? Reach out to us in autokit if you are looking for an automated solution to keep this table updated with OS releases. Could be helpful?

Reply:
I do not have a plan on how to keep the data updated. The OS team has talked about using the tool periodically to verify no regressions. This capability was another of the rationales behind the work.